db/migrate/20210113092719_add_unique_index_on_youth_risk_assessment.rb

```diff
@@ -0,0 +1,6 @@
+class AddUniqueIndexOnYouthRiskAssessment < ActiveRecord::Migration[6.0]
+  def change
+    remove_index :youth_risk_assessments, :profile_id
+    add_index :youth_risk_assessments, :profile_id, unique: true
```

Comment:
might be best to check for duplicates, otherwise this will fail

Reply:
good call, Ive checked on prod and staging, and there are no duplicates
```
YouthRiskAssessment.select(:profile_id).group(:profile_id).having("count(*) > 1")
=> #<ActiveRecord::Relation []>
```

Reply:
jolly good 👍